app/store/chat.ts

```diff
@@ -560,6 +561,10 @@ export const useChatStore = createPersistStore(
         const lastSummarizeIndex = session.messages.length;
 
         console.log("[Chat History] ", toBeSummarizedMsgs);
+        const lastMsgAssistant =
+          toBeSummarizedMsgs[toBeSummarizedMsgs.length - 1];
+        await searchVidHippo(lastMsgAssistant.content);
+        saveConentHippo(lastMsgAssistant.content);
```

Comment:
save to local storage? consider naming the function to reflect what it is doing.

Reply:
no, it save to database (P build)